src/Cpp/1-getting-started/1-HelloWindow/Application.cpp

```diff
@@ -22,9 +19,7 @@ void Application::Run()
 
     while (!glfwWindowShouldClose(_window))
     {
-        _input->Update(
-            static_cast<float>(_width) / 2.0f,
-            static_cast<float>(_height) / 2.0f);
+        _input->Update(static_cast<float>(_width) / 2.0f, static_cast<float>(_height) / 2.0f);
```

Comment:
ah here is a good example, I would like to have wraps when there are more than 1 method param, like it was on the left...
dunno if that is possible wif clang-format